pkg/events/parsers/parse_args.go

```diff
@@ -28,15 +22,15 @@ func ParseArgs(event *trace.Event) error {
 		arg.Value = ""
 	}
 
-	parseOrEmptyString := func(arg *trace.Argument, sysArg parsers.SystemFunctionArgument, err error) {
+	parseOrEmptyString := func(arg *trace.Argument, sysArg SystemFunctionArgument, err error) {
 		emptyString(arg)
 		if err == nil {
 			arg.Value = sysArg.String()
 		}
 	}
 
-	switch ID(event.EventID) {
-	case MemProtAlert:
+	switch event.EventName {
+	case "mem_prot_alert":
```

Comment:
We can't use the `events.ID` values because we can't import the `events` package.
If this package is imported by the signatures, and the signatures are used by Tracee, then it will cause a circular dependency to import Tracee for the events IDs.
The only solution is to have an external package that will translate the IDs to an informative value. I think that we agreed that it will be possible once we use the new event's structure and the protobuf values could be used.